mock-data/gameDetails.js

```diff
@@ -0,0 +1,9 @@
+const gameDetails = {
```

Comment:
I also made a mock game data file, looks almost identical so should be no problem merging them together